build.gradle

```diff
@@ -49,10 +49,9 @@ project.ext.externalDependency = [
     'jerseyGuava': 'org.glassfish.jersey.bundles.repackaged:jersey-guava:2.25.1',
     'jsonSimple': 'com.googlecode.json-simple:json-simple:1.1.1',
     'junit': 'junit:junit:4.12',
+    // avro-serde includes dependencies for `kafka-avro-serializer` `kafka-schema-registry-client` and `avro`
     'kafkaAvroSerde': 'io.confluent:kafka-streams-avro-serde:5.2.2',
     'kafkaClients': 'org.apache.kafka:kafka-clients:2.3.0',
-    'kafkaSchemaRegistry': 'io.confluent:kafka-schema-registry-client:3.3.1',
```

Comment:
Confluent libraries added support for schema registry basic auth in v4.1.0
https://docs.confluent.io/current/schema-registry/changelog.html#version-4-1-0

Since `kafkaAvroSerde` is already at `5.2.2` and it transitively includes both `kafka-avro-serializer` and `kafka-schema-registry-client` I decided to just drop these pinned dependencies entirely.

Reply:
Looking at this further, while this change fixes my issue with basic auth we may need to a higher version of the schema registry client to get SSL support for the schema registry client. 

https://docs.confluent.io/current/schema-registry/changelog.html#version-5-4-0

I unfortunately do not have an SSL enabled schema registry to test against so it is hard to validate.

Reply:
Hi @grantatspothero, did you include a higher version of the schema registry client to get SSL support as well?

Reply:
Sorry I merged this before @grantatspothero had a chance to bump the version up further. @themightylaz could you create a PR to further bump it up?

Reply:
I unfortunately do not have an SSL enabled schema registry to test with. If you have one and can test bumping the version higher that would be awesome.

Reply:
Bumped up in #1898.